### DISCUSSION TITLE
Schematic for v041 PCB

### DISCUSSION THREAD

**Discussion by vk5ln** [frntc/Sidekick64#17] [General]
I have (almost) finished building a v041 board and it is working very well. I am still waiting for a TFT display to arrive.

I'm wondering if the matching schematic for the v041 will be available? The PDF in the Gerber folder is for the older v03 gerbers.

**Comment by frntc** [comment]
True, I should update it at some point. Until then: not much has changed since then, the only additions are the A13<->Button jumper and the TFT screen which is connected to the same pins as the old ones plus the LED 3 and 4 signals (all come from the LVC573).

**Reply by vk5ln** [reply]
That would be good if it could be updated at some stage, no rush at all.

I'm having lots of fun with my SideKick64 :-)
